packages/ui-core/src/types.ts

```diff
@@ -12,3 +12,5 @@ export type Position = 'top' | 'bottom' | 'left' | 'right';
 export type Alignment = 'left' | 'center' | 'right';
 
 export type Layout = 'vertical' | 'horizontal';
+
+export type ColorModeType = 'steps' | 'shades';
```

Comment:
That type is duplicated across two packages - after renaming to `ColorMode`. I would suggest to leave it in `@keen.io/ui-core` then import it in charts.